Backend/app.py

```diff
@@ -38,7 +88,19 @@ def chat():
         if not message:
             return jsonify({"error": "Message cannot be empty"}), 400
 
-        response = openai_service.get_response(message)
+        # Get agent variant from feature management
+        agent_variant = feature_manager.get_variant("Agent")
```

Comment:
Before accessing the feature manager the config provider should be refreshed via the refresh api